include/swift/AST/DiagnosticsParse.def

```diff
@@ -1009,7 +1009,7 @@ ERROR(dollar_numeric_too_large,none,
 ERROR(numeric_literal_numeric_member,none,
       "expected named member of numeric literal", ())
 ERROR(standalone_dollar_identifier,none,
-     "'$' is not an identifier; use backticks to escape it", ())
+      "'$' is not an identifier; use backticks to escape it", ())
```

Comment:
Can these kind of indentation changes go in a different commit?  These have nothing to do with the change proposed in the pull request.